.meta/sources/file.toml

```diff
@@ -178,6 +178,88 @@ Instead of balancing read capacity fairly across all watched files, prioritize \
 draining the oldest files before moving on to read data from younger files.\
 """
 
+[sources.file.options.multiline]
+type = "table"
+category = "Multiline"
+common = true
+required = false
+description = """\
+Multiline parsing configuration. \
+If not speicified, multiline parsing is disabled.\
```

Comment:
Spelling issue.  Should be "specified."

Reply:
@rcollette That spelling error has been addressed in later code. Thanks for letting us know though!